### PR TITLE
Warn that renaming branches breaks actions

### PR DESCRIPTION
**Pull request by jsoref**
### Why:

If you rename a branch in a repository that hosts actions, workflows relying on those actions by branch name will break.

**Closes [issue link]**
image

### What's being changed:

Add text to explain this.
Screenshot_20210412-185952

### Check off the following:
- [x] I have reviewed my changes in staging. (look for the **deploy-to-heroku** link in your pull request, then click **View deployment**)
- [ ] For content changes, I have reviewed the [localization checklist](https://github.com/github/docs/blob/main/contributing/localization-checklist.md)
- [ ] For content changes, I have reviewed the [Content style guide for GitHub Docs](https://github.com/github/docs/blob/main/contributing/content-style-guide.md).

### CODE REVIEW

**Comment by janiceilene** [comment]
👋 @jsoref Thanks for opening a PR! Can you add more information to the **Why** and **What's being changed** sections of the pull request template in the OP to help ensure this is triaged properly? Thanks!

**Comment by jsoref** [comment]
@janiceilene ⚡

**Review by Nirzak — DISMISSED** [review]

File: content/github/administering-a-repository/renaming-a-branch.md [hunk 1]
```diff
@@ -15,6 +15,8 @@ You can rename a branch in a repository on {% data variables.product.product_loc
 
 When you rename a branch on {% data variables.product.product_location %}, any URLs that contain the old branch name are automatically redirected to the equivalent URL for the renamed branch. Branch protection policies are also updated, as well as the base branch for open pull requests (including those for forks) and draft releases. After the rename is complete, {% data variables.product.prodname_dotcom %} provides instructions on the repository's home page directing contributors to update their local Git environments.
 
+GitHub workflows do not follow renames, so if your repository publishes an action, anyone using that action with `@{old-branch-name}` will break. You should consider adding a new branch with the original content plus an additional commit reporting that the banch name is deprecated and suggesting that users migrate to the new branch name.
+
 Although file URLs are automatically redirected, raw file URLs are not redirected. Also, {% data variables.product.prodname_dotcom %} does not perform any redirects if users perform a `git pull` for the previous branch name.
```
Comment (lecoursen) [inline content/github/administering-a-repository/renaming-a-branch.md:20]:
```suggestion
Although file URLs are automatically redirected, raw file URLs are not redirected. Also, {% data variables.product.prodname_dotcom %} does not perform any redirects if users perform a `git pull` for the previous branch name.

{% data variables.product.prodname_actions %} workflows do not follow renames, so if your repository publishes an action, anyone using that action with `@{old-branch-name}` will break. You should consider adding a new branch with the original content plus an additional commit reporting that the banch name is deprecated and suggesting that users migrate to the new branch name.
```

**Review by lecoursen — DISMISSED** [review]
Thanks @jsoref! I added a suggestion to move the new sentence slightly and use a product name variable, but I'll apply that suggestion and merge now!

File: content/github/administering-a-repository/renaming-a-branch.md (hunk 1 above)
Comment (jsoref) [reply content/github/administering-a-repository/renaming-a-branch.md]: @lecoursen: could you add an action or test which complains about bare instances of the things you don't want used bare and suggests the replacements?

(I had a tool a long time ago that could do things like that)

It'd really help. It's much better than asking random contributors to memorize or somehow divine the table of such things.

**Comment by jsoref** [comment]
@lecoursen: can you please make this stop? (i.e. please send feedback to someone who will listen that they are dissuasive wrt external contributions)
 
image

It's really confusing and really unhelpful. Speaking as someone who is trying to make contributions to your project. (And many others.) I am now getting regularly haunted by these notifications.

I'm actually quite technical. But whether I wear a non-technical doc writing hat, or a technical code fixing hat, these notifications do not do me any good.

* They aren't actionable
* They are all sorts of red, which in normal terms means I should drop what I'm doing and do something to fix them. But I can't.
* They use a ⚠️ icon which should be 🟡 and paint it 🔴  (specifically 🔺). It's true that drives may face a red yield icon, but not everyone drives, and why in the world would you use that iconography in this context?

**Comment by jsoref** [comment]
Fwiw, it looks like the most actionable action I can take here would be to click the shiny `[ Disable auto-merge ]` button. Which I'd be tempted to do if it would make the notifications go away.

**Review by lecoursen — APPROVED** [review]

**Comment by janiceilene** [comment]
Oh my gosh @jsoref I'm so sorry about all of those notifications! The new "enable workflows" process was fighting with automerge on this one. Hopefully everything we'll get it all smoothed out soon, but in the meantime, I'll get this feedback to the right internal team. 

Thank so much for your contribution! We really appreciate you sticking with us through this 💖

**Pull request merged**